ts_scripts/install_utils

```diff
@@ -28,9 +28,9 @@ install_java_deps()
 
```

Comment:
I see this being used in only two [places](https://github.com/pytorch/serve/search?q=%22install_utils%22). Do we still need this or can we migrate it to one of the python util scripts

Reply:
This has been referenced in Circle CI. I think we can move out the circle ci related code and this script to a different PR. This file doesn't affect the current installation scripts. No changes required in existing python utils.